releasenotes/config.yaml

```diff
@@ -2,7 +2,15 @@
 encoding: utf8
 default_branch: main
 sections:
-  - [features, New Features]
+  - [features, New Features, 1]
+  - [features_expnew, New Experiments, 2]
+  - [features_expupdate, Experiment Updates, 2]
+  - [features_expclass, Experiment Class Features, 2]
+  - [features_analysis, Analysis Class Features, 2]
+  - [features_expdata, Experiment Data Features, 2]
```

Comment:
I tested with #1342 merged in. Just a note that its release note needs to be updated to go under a section after it is merged (assuming it merges before this PR).

Reply:
Will do. I have to move the note to the `0.6/` folder too so this PR should be the last one merged before release.